cli/tests/unit/ffi_test.ts

```diff
@@ -29,7 +29,8 @@ Deno.test({ permissions: { ffi: false } }, function ffiPermissionDenied() {
     Deno.dlopen("/usr/lib/libc.so.6", {});
   }, Deno.errors.PermissionDenied);
   const fnptr = new Deno.UnsafeFnPointer(
-    0n,
+    // @ts-expect-error: Not NonNullable but null check is after premissions check.
```

Comment:
"Not NonNullable" => "can be null"

Reply:
hmm actually why can't it be `null` and pass type checking?

Reply:
As `UnsafeFnPointer` is only meant for calling C functions, its pointer can never be `null` and the call to still work. On the Rust side an error will be thrown on null pointer, and libffi might even panic if we let them through.